MyPerfectOnboardingApplication/Tests/MyPerfectOnboarding.Tests.Utils/Comparers/ListItemEqualityComparer.cs

```diff
@@ -1,20 +1,27 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using MyPerfectOnboarding.Contracts.Models;
 
-namespace MyPerfectOnboarding.Api.Tests.Utils
+namespace MyPerfectOnboarding.Tests.Utils.Comparers
 {
-    public sealed class ListItemEqualityComparer : IEqualityComparer<ListItem>
+    public sealed class ListItemEqualityComparer : IEqualityComparer<IListItem>
     {
-        public bool Equals(ListItem x, ListItem y)
+        private static readonly Lazy<ListItemEqualityComparer> LazyComparer = new Lazy<ListItemEqualityComparer>(() => new ListItemEqualityComparer());
+
+        public static IEqualityComparer<IListItem> Instance => LazyComparer.Value;
+
+        private ListItemEqualityComparer() { }
+
+        public bool Equals(IListItem x, IListItem y)
         {
             if (ReferenceEquals(x, y)) return true;
             if (ReferenceEquals(x, null)) return false;
             if (ReferenceEquals(y, null)) return false;
-            if (x.GetType() != y.GetType()) return false;
+            //if (x.GetType() != y.GetType()) return false;
             return x.Id.Equals(y.Id) && string.Equals(x.Text, y.Text) && x.IsActive == y.IsActive && x.CreationTime.Equals(y.CreationTime) && x.LastUpdateTime.Equals(y.LastUpdateTime);
```

Comment:
```suggestion
            return x.Id.Equals(y.Id)
                && string.Equals(x.Text, y.Text)
                && x.IsActive == y.IsActive
                && x.CreationTime.Equals(y.CreationTime)
                && x.LastUpdateTime.Equals(y.LastUpdateTime);
```
Original code looks aweful 👼